Process.h

```diff
@@ -84,6 +112,12 @@ typedef struct Process_ {
    /* Foreground group identifier of the controlling terminal */
    int tpgid;
 
+   /* This is a kernel (helper) task */
+   bool isKernelThread;
+
+   /* This is a userland thread / LWP */
+   bool isUserThread;
+
    /*
```

Comment:
Could these two and the procExeDeleted field be single-bit bitfields?  (as done in ProcessMergedCommand)

Reply:
Sure they could. Maybe also combine with the fields `updated`, `tag`, `show`, `wasShown`, and `showChildren` below.